README.md

```diff
@@ -1,17 +1,22 @@
 # install-phylum-latest-action
 A GitHub Action to install the latest version of the Phylum [command-line tool](https://github.com/phylum-dev/cli).
 
-This action enables users to download, install, and configure the Phylum command-line interface (CLI) tool for use. The Phylum CLI tool allows users to submit their project package dependencies to Phylum's API for analysis. 
-
-Phylum provides a complete risk analyis of "open-source packages" (read: untrusted software from random Internet strangers). Phylum evolved forward from legacy SCA tools to defend from supply-chain malware, malicious open-source authors, and engineering risk, in addtion to software vulnerabilities and license risks. To learn more, please see [our website](https://phylum.io).
-
+This action enables users to download, install, and configure the Phylum command-line interface (CLI) tool for use.
+The Phylum CLI tool allows users to submit their project package dependencies to Phylum's API for analysis.
 
+Phylum provides a complete risk analyis of "open-source packages" (read: untrusted software from random Internet
+strangers). Phylum evolved forward from legacy SCA tools to defend from supply-chain malware, malicious open-source
+authors, and engineering risk, in addition to software vulnerabilities and license risks. To learn more, please see
+[our website](https://phylum.io).
 
 ## Features
 - can be used by other GitHub Actions to set Phylum up in the environment
 
 ## Getting Started
-This is a sample workflow using this action. Note the `export` to add the phylum install directory to your `PATH`.
+This is a sample workflow using this action.
+<!-- TODO: Remove the export line here and in the action? It appears to already be happening in the CLI installer -->
```

Comment:
I didn't get a chance to try this...but might still if I can get the `TestGHA` repo working on my local system.

Reply:
Didn't get to it...but it's not really hurting anything...